lib/svg-to-vectordrawable.js

```diff
@@ -597,53 +597,61 @@ JS2XML.prototype.round = function(value, floatPrecision) {
     return Math.round(value * Math.pow(10, floatPrecision)) / Math.pow(10, floatPrecision);
 };
 
-JS2XML.prototype.convert = function(data, floatPrecision) {
+JS2XML.prototype.convert = function(data, floatPrecision, strict) {
```

Comment:
I wonder if it would be better to switch to a configuration object, instead of adding more and more function parameters.